polaris/ocean/model/mpaso_to_omega.yaml

```diff
@@ -3,14 +3,26 @@ dimensions:
   nCells: NCells
   nEdges: NEdges
   nVertices: NVertices
+  nTracers: NTracers
   maxEdges: MaxEdges
   maxEdges2: MaxEdges2
   TWO: MaxCellsOnEdge
   vertexDegree: VertexDegree
-  nVertLevels: NVertLevels
-  nVertLevelsP1: NVertLevelsP1
+  # Don't map nVertLevels yet -- needs Omega fix
+  # nVertLevels: NVertLevels
+  # nVertLevelsP1: NVertLevelsP1
```

Comment:
If you decide to merge before the Omega fix, please change to nVertLayers*.

Reply:
But the purpose here was literally just to comment out what was there before.  I don't see a lot of added value in fixing it here and leaving it commented out.  Let's just fix it as part of commenting it back in.